frontend/utils/time.js

```diff
@@ -43,3 +43,13 @@ export function compareISOTimestamps (a: string, b: string): number {
   const B = new Date(b).getTime()
   return A > B ? 1 : (A < B ? -1 : 0)
 }
+
+// TODO: Provide locale fallback in case navigator does not exist (e.g. server, Mocha, etc...)
+const locale = (navigator && navigator.languages ? navigator.languages[0] : navigator.language) || 'en-US'
```

Comment:
Oops, it's still failing. It took me a bit but I finally figured out how to write this so that it doesn't fail:

```js
const locale = (typeof navigator === 'undefined' && 'en-US') || (navigator.languages ? navigator.languages[0] : navigator.language)
```
Verified locally that that works!